CONTRIBUTING.md

```diff
@@ -136,7 +136,8 @@ If you can, try to reproduce the highlighting problem in the text editor that th
 
 You can also try to fix the bug yourself and submit a pull-request.
 [TextMate's documentation](https://manual.macromates.com/en/language_grammars) offers a good introduction on how to work with TextMate-compatible grammars.
-You can test grammars using [Lightshow](https://github-lightshow.herokuapp.com).
+Note that Linguist uses [PCRE](https://www.pcre.org/) regular expressions, while TextMate uses [Oniguruma](https://github.com/kkos/oniguruma) library. Although they are mostly compatible there might be some differences in syntax and semantics between the two. 
+You can test grammars against the GitHub’s syntax highlighter using [Lightshow](https://github-lightshow.herokuapp.com).
```

Comment:
Any particular reason the original wording of this sentence was changed?

Reply:
I wanted to stress that the `Lightshow` is going to validate grammar specifically against the github engine, as there might be other TextMate compatible syntax editors online. It becomes more or less clear when user visits `Lightshow` page, but I thought it's a good idea to highlight this in the document.

Reply:
Well, the proposed wording isn't adding anything which isn't already inferred by context. It's better we keep the readme as clear and concise as possible, which means eliminating unnecessary verbiage like this.